release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -404,6 +404,50 @@ In {product-title} {product-version} and later, you can edit the baseboard manag
 [id="ocp-4-16-monitoring_{context}"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-16-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.12.0
+* Metrics Server to 0.7.1
+* node-exporter to 1.8.0
+* Prometheus to 2.52.0
+* Prometheus Operator to 0.73.2
+* Thanos to 0.35.0
+
+[id="ocp-4-16-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `ClusterMonitoringOperatorDeprecatedConfig` alert to monitor when the Cluster Monitoring Operator configuration uses a deprecated field.
+** Added the `PrometheusOperatorStatusUpdateErrors` alert to monitor when the Prometheus Operator fails to update object status.
+
+[id="ocp-4-16-monitoring-metrics-server-to-access-metrics-api-ga"]
+==== Metrics Server component to access the Metrics API general availability (GA)
+
+Metrics Server component is now generally available and automatically installed instead of the deprecated Prometheus Adapter. Metrics Server collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality. For more information, see xref:../observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-16-monitoring-new-role-to-allow-read-only-access-to-alertmanager-api"]
+==== New monitoring role to allow read-only access to the Alertmanager API
+
+This release introduces a new `monitoring-alertmanager-view` role to allow read-only access to the Alertmanager API in the `openshift-monitoring` project.
+
+[id="ocp-4-16-monitoring-vpa-metrics-available-in-kube-state-metrics-agent"]
+==== VPA metrics are available in OpenShift kube-state-metrics agent
```

Comment:
```suggestion
==== VPA metrics are available in the `kube-state-metrics` agent
```

Reply:
Based on OpenShift guidelines, we are not supposed to use backticks in headings, that should imply for RN titles, if I am not mistaken. I see it like this in the existing examples:
* "Introduction of HAProxy no strict-limits variable" ([link](https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-notable-technical-changes))
Instead of "Introduction of HAProxy no `strict-limits` variable"
Or "Must-gather storage limit default" instead of "`must-gather` storage limit default"

So I am retaining the existing version (I will fix the article though).